uaclient/messages/__init__.py

```diff
@@ -1534,13 +1534,14 @@ def __repr__(self):
 the --access-only flag.""",
 )
 
-UNEXPECTED_ERROR = NamedMessage(
+UNEXPECTED_ERROR = FormattedNamedMessage(
     "unexpected-error",
     t.gettext(
         """\
-Unexpected error(s) occurred.
-For more details, see the log: /var/log/ubuntu-advantage.log
-To file a bug run: ubuntu-bug ubuntu-advantage-tools"""
+Unexpected error(s) occurred: {error_msg}
```

Comment:
I liked the idea of asking the user to look for the error first before opening a bug, but I am still unsure if we should say that directly. What do you think @orndorffgrant @renanrodrigo ?

Reply:
That is a suggested requirement of bug #2600 
I'd say it is good to tell users to investigate first, at least to say we tried hah
Not sure how effective it will really be, but I don't think it hurts.